solutions/swb-app/src/errors/noKeyExistsError.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+export class NoKeyExistsError extends Error {
+  public readonly isNoKeyExistsError: boolean;
+
+  public constructor(message?: string) {
+    super(message);
+
+    // Maintains proper stack trace for where our error was thrown (only available on V8)
```

Comment:
We should remove all the copypasta into a `CustomError` class to minimize duplication, but don't its definitely not a blocker for this PR.

Reply:
I'm going to make a follow up ticket for this! Thanks for the idea.